sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/Sample08_Administration.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Identity;
+using NUnit.Framework;
+using Azure.Messaging.ServiceBus.Administration;
+
+namespace Azure.Messaging.ServiceBus.Tests.Samples
+{
+    public class Sample08_Administration : ServiceBusLiveTestBase
+    {
+        /// <summary>
+        /// Authenticate with <see cref="DefaultAzureCredential"/>.
+        /// </summary>
+        public void AuthenticateWithConnectionString()
```

Comment:
Let's also add one for authenticating with AAD for good measure (similar to the sample for ServiceBusClient).